.circleci/config.yml

```diff
@@ -17,12 +17,34 @@ jobs:
           path: "src/label/test-images"
       - store_artifacts:
           path: "src/resize/test-images"
+  test-node8:
+    <<: *test-common
+    docker:
+      - image: dwmkerr/node:8-imagemagick
+  test-node10:
+    <<: *test-common
+    docker:
+      - image: dwmkerr/node:10-imagemagick
+  test-node12:
+    <<: *test-common
+    docker:
+      - image: dwmkerr/node:12-imagemagick
 
 workflows:
   version: 2
   build:
     jobs:
-      - test:
+      - test-node8:
```

Comment:
Will these run in parallel out of curiosity?

Reply:
Yes they will - from memory there is config to determine dependencies between jobs

Reply:
Thanks, just noticed the same thing.